From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: davidchao <davidchao@google.com>
Date: Fri, 30 Oct 2020 22:18:02 +0800
Subject: ANDROID: GKI: enable CONFIG_THERMAL_NETLINK

Bug: 170682696
Test: boot to home and thermal_genl_send_event worked properly.
Change-Id: I3764f6f8eaa47eab6f0941617bf6268bb3b4db4d
Signed-off-by: davidchao <davidchao@google.com>
(cherry picked from commit e8173a7565113b7ccdb32a9719f7ab028e070b02)
Signed-off-by: Will McVicker <willmcvicker@google.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@google.com>
---
 arch/arm64/configs/gki_defconfig | 1 +
 arch/x86/configs/gki_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -341,6 +341,7 @@ CONFIG_GPIO_GENERIC_PLATFORM=y
 CONFIG_POWER_RESET_HISI=y
 # CONFIG_HWMON is not set
 CONFIG_THERMAL=y
+CONFIG_THERMAL_NETLINK=y
 CONFIG_THERMAL_STATISTICS=y
 CONFIG_THERMAL_EMERGENCY_POWEROFF_DELAY_MS=100
 CONFIG_THERMAL_WRITABLE_TRIPS=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -300,6 +300,7 @@ CONFIG_SPI=y
 CONFIG_GPIOLIB=y
 CONFIG_GPIO_GENERIC_PLATFORM=y
 # CONFIG_HWMON is not set
+CONFIG_THERMAL_NETLINK=y
 CONFIG_THERMAL_STATISTICS=y
 CONFIG_THERMAL_EMERGENCY_POWEROFF_DELAY_MS=100
 CONFIG_THERMAL_WRITABLE_TRIPS=y
